helm_deploy/values-preprod.yaml

```diff
@@ -4,9 +4,8 @@ generic-service:
   replicaCount: 1
 
   ingress:
-    enable_whitelist: false
     hosts:
-      - host: calculate-journey-variable-payments-preprod.apps.cloud-platform.service.justice.gov.uk
+      - calculate-journey-variable-payments-preprod.apps.cloud-platform.service.justice.gov.uk
     aws_weight: "100"
```

Comment:
these references to `aws_weight` don't do anything and can be removed

Reply:
sorted